android/KMEA/app/src/main/assets/keyboard.html

```diff
@@ -248,7 +248,8 @@
         var keyPos = x.toString() + ',' + y.toString();
         for(i=0; i<obj.length; i++)
         {
-          s=s+obj[i].layer+'-'+obj[i].coreID;
+          // elementID contains the layer and coreID
+          s=s+obj[i].elementID;
```

Comment:
I don't understand why we are making this change? Is this a separate fix bundled in?

Reply:
It's from #5651 that corresponds @jahorton 's DM to me about activeLayout.ts
https://github.com/keymanapp/keyman/blob/6c17ecbd5f3f4ccd47d2330944a513125d06af1a/common/core/web/keyboard-processor/src/keyboards/activeLayout.ts#L112-L118

> This should be s=s+obj[i].elementID .